molecule/default/destroy.yml

```
@@ -63,8 +63,8 @@
     - name: Upload logs to object storage
       ignore_errors: True
       when:
-        - lookup('env', 'GITHUB_RUN_ID') is defined
-        - lookup('env', 'GITHUB_RUN_NUMBER') is defined
+        - not lookup('env', 'GITHUB_RUN_ID') == ""
+        - not lookup('env', 'GITHUB_RUN_NUMBER') == ""
```

@mpiscaer Thanks for your contribution.
I suggest you can follow `ComparisonToEmptyStringRule`
https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/empty_string_compare.py#L18-L25

```suggestion
        - lookup('env', 'GITHUB_RUN_ID')|length > 0
        - lookup('env', 'GITHUB_RUN_NUMBER')|length > 0
```